bin/whisper-resize.py

```diff
@@ -132,11 +123,12 @@
 
   new_info = whisper.info(newfile)
   new_archives = new_info['archives']
+  new_archives.sort(key=lambda a: a['secondsPerPoint'], reverse=True)
 
   for archive in new_archives:
     step = archive['secondsPerPoint']
-    fromTime = now - archive['retention'] + now % step
-    untilTime = now + now % step + step
+    fromTime = now - archive['retention'] - now % step
+    untilTime = now - now % step + step
```

Comment:
It's unclear to me why you need this change, could you explain it in the commit msg ?